website/siteConfig.js

```diff
@@ -22,12 +22,126 @@ embed.register('stylesheet',
+        pinned: true,
+    },
+    {
+        caption: 'Editor.md',
+        image: 'https://pandao.github.io/editor.md/images/logos/editormd-logo-180x180.png',
+        infoLink: 'https://pandao.github.io/editor.md/en.html',
+        pinned: true,
+    },
+    {
+        caption: 'Expii',
+        image: 'https://78.media.tumblr.com/avatar_ccde3e53e98f_128.pnj',
+        infoLink: 'https://www.expii.com/',
+        pinned: true,
+    },
     {
         caption: 'GitLab',
         image: 'https://gitlab.com/gitlab-com/gitlab-artwork/raw/master/logo/logo-square.png',
         infoLink: 'https://gitlab.com/',
         pinned: true,
     },
+    {
+        caption: 'Gitter',
+        image: 'https://assets.gitlab-static.net/uploads/-/system/project/avatar/3601513/gitter_logo.png',
+        infoLink: 'https://gitter.im/',
+        pinned: true,
+    },
+    {
+        caption: 'Gatsby',
+        image: 'https://www.gatsbyjs.org/monogram.svg',
+        infoLink: 'https://www.gatsbyjs.org/',
+        pinned: true,
+    },
+    {
+        caption: 'Gradescope',
+        image: 'https://pbs.twimg.com/profile_images/920085750095298560/AqGnlpiJ_400x400.jpg',
+        infoLink: 'https://www.gradescope.com/',
+        pinned: true,
+    },
+    {
+        caption: 'hack.chat',
+        image: 'https://hack.chat/apple-icon-180x180.png',
+        infoLink: 'https://hack.chat/',
+        pinned: true,
+    },
+    {
+        caption: 'Idyll',
+        image: 'https://cdn.rawgit.com/idyll-lang/idyll-lang.github.io/src/images/logo.svg',
+        infoLink: 'https://idyll-lang.org/',
+        pinned: true,
+    },
+    {
+        caption: 'Messenger',
+        image: 'https://en.facebookbrand.com/wp-content/uploads/2016/09/messenger_icon2.png',
+        infoLink: 'https://www.messenger.com/',
+        pinned: true,
+    },
+    {
+        caption: 'Observable',
+        image: 'https://pbs.twimg.com/profile_images/970805785503477760/HfTZJiZo_400x400.jpg',
+        infoLink: 'https://beta.observablehq.com/',
+        pinned: true,
+    },
+    {
+        caption: 'Quill',
+        image: 'https://quilljs.com/assets/images/logo.svg',
+        infoLink: 'https://quilljs.com/',
+        pinned: true,
+    },
+    {
+        caption: 'Rocket.Chat',
+        image: 'https://cdn.rawgit.com/RocketChat/Rocket.Chat.Artwork/master/Logos/icon.svg',
+        infoLink: 'https://rocket.chat/',
+        pinned: true,
+    },
+    {
+        caption: 'Slides',
+        image: 'https://s3.amazonaws.com/uploads.uservoice.com/logo/design_setting/116173/original/slides-symbol-150x150.png',
+        infoLink: 'https://slides.com/',
+        pinned: true,
+    },
+    {
+        caption: 'Spinning Numbers',
+        image: 'https://spinningnumbers.org/i/sn_logo2.svg',
+        infoLink: 'https://spinningnumbers.org/',
+        pinned: true,
+    },
+    {
+        caption: 'StackEdit',
+        image: 'https://cdn.rawgit.com/benweet/stackedit/master/static/landing/favicon.ico',
```

Comment:
High resolution logo seems to be available in https://github.com/benweet/stackedit/blob/master/src/assets/iconStackedit.svg and https://github.com/benweet/stackedit/blob/master/src/assets/favicon.png.